BLSQ/iaso
# IA-1976: Enable personalized user guide per IASO account

Explain what problem this PR is resolving
- For each acccount, only IASO user manual was accessible even for POLIO account
- This ticket is to enable personalized user guide per account
Related JIRA tickets : [IA-1976](https://bluesquare.atlassian.net/browse/IA-1976)

## Self proofreading checklist


## Changes

- Create a migration to add user_manual_path to account
- Add user_manual_path field to account as_dict method 
- On the front check if the account user_manual_path is null, if yes display the default user manual if not display the account user manual path

## How to test
- Edit an account by adding a personalized user guide link
- Go on the bottom of the sidebar and check if the user manual is the one added on the account


[IA-1976]: https://bluesquare.atlassian.net/browse/IA-1976?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
